File: JavaScript - Part II/JavaScript APIs/5. MovingCircle/scripts/point.js
var Point = (function () {
    var Point = function (x, y) {
        this.x = x;
        this.y = y;
    };

    Point.add = function (point1, point2) {
        return new Point(
            point1.x + point2.x,
            point1.y + point2.y
        );
    };

    Point.multiply = function (number, point) {
        return new Point(number * point.x, number * point.y);
    };

    Point.prototype.toString = function () {
        var pointAsString = "x: " + this.x + ", y: " + this.y;

        return pointAsString;
    };

    return Point;
}());

File: Data Structures and Algorithms/Other Algorithms/2. PointInsideTriangle/Point.cs
using System;

public struct Point
{
    public Point(double x, double y, double z)
        : this()
    {
        this.X = x;
        this.Y = y;
        this.Z = z;
    }

    public double X { get; set; }

    public double Y { get; set; }

    public double Z { get; set; }
}